indexer/src/main.ts

```diff
@@ -33,7 +33,15 @@ const sinkOptions =
           Deno.env.get("MONGO_CONNECTION_STRING") ??
           "mongodb://mongo:mongo@mongo:27017",
         database: Deno.env.get("MONGO_DATABASE_NAME") ?? "kakarot-test-db",
-        collectionNames: ["headers", "transactions", "receipts", "logs"],
+        collectionNames: [
+          "headers",
+          "transactions",
+          "receipts",
+          "logs",
+          "transactions_failure",
+          "receipts_failure",
+          "logs_failure",
```

Comment:
I don't think it makes sense to index receipts and logs failures, since the `out of resources` error will happen in the middle of the transactions, meaning you are indexing half receipts/logs